### ISSUE TITLE
Feat: Add a "custom login page ready" custom event

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#4605]
## Feature description and goals

The custom login page is currently embedded in the HTML code of tracim, and then copied by the app into its React Login.jsx component. We shall have a custom event triggered when this copy is effective, so scripts in the custom login page can fiddle with these elements.

It is to be noted that this can happen several times if the user disconnects and reconnect. 

The event should be called `TRACIM_CUSTOM_LOGIN_PAGE_MOUNTED` so it can be listened using this snippet:

```
document.addEventListener('appCustomEventListener', function (event) {
  if (event.detail.type === 'TRACIM_HEADER_MOUNTED') {
    alert('Hello world!')
  }
})
```

**Comment by raphj** [comment]
I wonder if we should simplify a bit and allow users to simply write:

```js
document.addEventListener('customLoginPageReady', function (event) {
  alert('Hello world!')
})
```

**Comment by Skylsmoi** [comment]
No, this would create an ever expanding different custom event number and would become more and more difficult to maintain

It is easier to only listen to one custom event and trigger different function depending on its property "type", it rationalize the code in controller like functions instead of multiplying addEventListener in the code base

**Comment by lebouquetin** [comment]
Another interest is the ability to listen to all tracim events (which are actually only one)
